SUMMARY.md

```diff
@@ -0,0 +1,8 @@
+## Telliot Documentation
+* [Introduction](docs/README.md)
+* [Setup and usage](docs/01_setup-and-usage.md)
+* [Configuation](docs/02_configuation.md)
+* [Contributing](docs/03_contributing.md)
+* [Coding style guide](docs/04_coding-style-guide.md)
+* [Release process](docs/05_release-process.md)
+* [Changelog](docs/CHANGELOG.md)
```

Comment:
Can't we put this in the `docs` folder and then just use this file to merge in the main docs repo summary.md.

And if that is possible then we can remove the number suffex from the filenames.

Reply:
Yeah, possible. I'll make the required change to the main docs repo as well